### PR TITLE
[C-2365] Update play buttons on web and mobile to show resume when track is current

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Update the play buttons on web and mobile to display resume the track is completed, but is still the current track

### Dragons

N/A

### How Has This Been Tested?

Manually tested

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

File: packages/mobile/src/components/details-tile/DetailsTile.tsx [hunk 1]
```diff
@@ -220,6 +222,8 @@ export const DetailsTile = ({
   const navigation = useNavigation()
 
   const currentUserId = useSelector(accountSelectors.getUserId)
+  const currentPlayerTrackId = useSelector(getTrackId)
+  const isCurrentTrack = track && track.track_id === currentPlayerTrackId
```
Comment (sliptype) [inline packages/mobile/src/components/details-tile/DetailsTile.tsx:226]: Can you put this equality check into the selector, that will prevent extra rerenders of the details tile when the playing track changes:

```suggestion
  const isCurrentTrack = useSelector((state) => {
    return track && track.track_id === getTrackId(state)
 }
```

**Review by sliptype — APPROVED** [review]
Makes sense!

File: packages/mobile/src/components/details-tile/DetailsTile.tsx (hunk 1 above)
Comment (sliptype) [reply packages/mobile/src/components/details-tile/DetailsTile.tsx:226]: Also same deal on web

Comment (Kyle-Shanks) [reply packages/mobile/src/components/details-tile/DetailsTile.tsx:226]: 🧠

**Pull request merged**